ts-obs/cmd/timescaledbConnect.go

```diff
@@ -0,0 +1,133 @@
+package cmd
+
+import (
+    "errors"
+    "os"
+    "time"
+
+	"github.com/spf13/cobra"
+
+    corev1 "k8s.io/api/core/v1"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// timescaledbConnectCmd represents the timescaledb connect command
+var timescaledbConnectCmd = &cobra.Command{
+	Use:   "connect",
+	Short: "Connects to the TimescaleDB/PostgreSQL database",
+	RunE:  timescaledbConnect,
+}
+
+func init() {
+	timescaledbCmd.AddCommand(timescaledbConnectCmd)
+    timescaledbConnectCmd.Flags().StringP("user", "u", "postgres", "user to login with")
+    timescaledbConnectCmd.Flags().StringP("password", "p", "", "environment variable where password is stored")
+    timescaledbConnectCmd.Flags().BoolP("master", "m", false, "directly execute session on master node")
+}
+
+func timescaledbConnect(cmd *cobra.Command, args []string) error {
+    var err error
+
+    if len(args) != 0 {
+        return errors.New("\"ts-obs timescaledb connect\" requires 0 arguments")
+    }
+
+    var user string
+    user, err = cmd.Flags().GetString("user")
+    if err != nil {
+        return err
+    }
+
+    var password string
+    password, err = cmd.Flags().GetString("password")
+    if err != nil {
+        return err
+    }
+
+    var master bool
+    master, err = cmd.Flags().GetBool("master")
+    if err != nil {
+        return err
+    }
+
+    if password == "" == !master {
```

Comment:
We need exactly one of them to be true, since connecting through master automatically logs you in as postgres. Maybe change to an XOR to make that clearer.